src/data/data_store.cairo

```diff
@@ -691,6 +722,13 @@ mod DataStore {
             self.i128_values.write(key, Default::default());
         }
 
+        fn apply_delta_to_i128(ref self: ContractState, key: felt252, value: u128) -> u128 {
+            let next_int: u128 = self.i128_values.read(key) + value;
```

Comment:
can you harmonize and do the same way in `apply_bounded_delta_to_u128` please ? do the read and addition in one line

Reply:
since we have to return the result of the addition, isn't it more optimal/preferable to initialize the variable and then return it?